photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -285,8 +284,7 @@ class GameEstimator(val sc: SparkContext, val params: GameParams, implicit val l
       dataSets: Map[String, DataSet[_ <: DataSet[_]]],
       trainingEvaluator: Evaluator,
       validationDataAndEvaluators: Option[(RDD[(Long, GameDatum)], Seq[Evaluator])],
-      normalizationContexts: Option[Map[FeatureShardId, NormalizationContext]])
-    :Seq[(GameModel, Option[EvaluationResults], GameModelOptimizationConfiguration)] = {
+      normalizationContexts: Option[Map[FeatureShardId, NormalizationContext]]): Seq[GameResult] = {
 
     val contextBroadcasts: Option[Map[FeatureShardId, Broadcast[NormalizationContext]]] = normalizationContexts.map {
       contextsMap => contextsMap.mapValues { context => sc.broadcast(context) }
```

Comment:
How does the current implementation handle multiple configurations which don't touch the reg weight, but instead mess around with the optimizer or the type of regularization? For an example case of this with 1 fixed and 1 random effect, there would be 4 observations but each with the same hyperparameter vector.

I suppose that's my real question, so unrelated to this PR: does that make sense? Do any users do that? If not, we should change the way the configurations work so instead they have a list of lambdas to do grid search for before tuning.